include/libdnf/conf/option_seconds.hpp

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright (C) 2018-2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_CONF_OPTION_SECONDS_HPP
+#define LIBDNF_CONF_OPTION_SECONDS_HPP
+
+#include "option_number.hpp"
+
+namespace libdnf {
+
+/// Option that stores an integer value of seconds.
+/// It supports default value.
+/// It supports limits for minimal and maximal value.
```

Comment:
It could be replaced by:
`/// Support default value, minimal and maximal values.`